src/test/java/at/porscheinformatik/sonarqube/licensecheck/gradle/GradleProjectResolver.java

```diff
@@ -0,0 +1,46 @@
+package at.porscheinformatik.sonarqube.licensecheck.gradle;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.attribute.PosixFilePermissions;
+
+public class GradleProjectResolver {
+
+    private static final String DEFAULT_GRADLE_VERSION = "5.1.1";
+    private static final Logger LOGGER = LoggerFactory.getLogger(GradleProjectResolver.class);
+
+    public static void loadGradleWrapper(File projectRoot) throws IOException {
+        loadGradleWrapper(projectRoot, DEFAULT_GRADLE_VERSION);
+    }
+
+    public static void loadGradleWrapper(File projectRoot, String version) throws IOException {
+        String[] command = {projectRoot.getAbsolutePath() + "/gradlew", "wrapper", "--gradle-version=" + version};
+        ProcessBuilder processBuilder = new ProcessBuilder();
+        Process process = processBuilder.command(command).directory(projectRoot).start();
+        while (process.isAlive()) {
```

Comment:
Use Process#waitFor instead of busy wait and you will also get the exit value:
```suggestion
        if (process.waitFor() != 0) {
```